products/nutanix-aos.md

```diff
@@ -190,7 +190,6 @@ Cluster Check (NCC). Only the latest patch release in any given release cycle is
 
 ## Release Cadence
 
-* Major/Minor releases are typically made available every 3-6 months for STS Releases and every
-  12-15 months for LTS Releases.
+* Major/Minor releases are typically made available every 6-9 months.
```

Comment:
STS / LTS does not exist anymore, so the "Release" paragraph above must be updated accordingly.

Reply:
File https://github.com/endoflife-date/endoflife.date/issues/7405 for it.